scapy/utils.py

```diff
@@ -933,12 +1045,13 @@ def corrupt_bytes(s, p=0.01, n=None):
 
 
 @conf.commands.register
-def corrupt_bits(s, p=0.01, n=None):
+def corrupt_bits(x, p=0.01, n=None):
```

Comment:
It will be better to replace s by string and avoid single letter variables wherever we can.

Reply:
I like the `x` variable when it comes to util functions. It can also be either a string or bytes, so I think it would be confusing..

Reply:
Done